packages/ui/src/components/select/MultiProxySelection.tsx

```diff
@@ -27,7 +28,7 @@ const isOptionEqualToValue = (option?: MultiProxy, value?: MultiProxy) => {
   return option.multisigs[0].address === value.multisigs[0].address
 }
 
-const MultiProxySelection = ({ className }: Props) => {
+const MultiProxySelection = ({ className, isDesktop }: Props) => {
```

Comment:
I would probably use the hook, to avoid passing the props to the component. F.e
```ts
const isDesktop = useMediaQuery((theme: Theme) => theme.breakpoints.up('md'))
```

Reply:
I don't think this hook should be used here. At the end of the day, it's just a `testId` to identify the specific `MultiProxySelection` component. The fact that we're using it for dektop or mobile is not relevant.

I'll rename it in `testId`.